releasenotes/notes/otel-mapping-go-0.2.0-c80b8b4dcb8cf680.yaml

```
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Use literal 'host' resource attribute on OTLP payloads when present as the host value to avoid double tagging.
```

Non-blocker sentence edit 
```suggestion
    When present, use 'host' resource attribute as the host value on OTLP payloads to avoid double tagging.
```